FUNDING.md

```diff
@@ -23,6 +23,7 @@ _List borrowed from [ESLint: "Funding ESLint's Future."](https://eslint.org/blog
 100% of money raised is invested back into the community. Every dollar spent **must** support and/or improve Astro in some way. For example:
 
 - **Swag!** Creating stickers, t-shirts, etc. for sponsors and community members.
```

Comment:
@sarah11918 pointed out the other day that this is an odd thing to list first. Maybe we should move it down or even remove it?